frontend/src/state/user/userEpics.js

```diff
@@ -0,0 +1,34 @@
+// RXJS OPERATORS
+import { catchError, mergeMap, map } from 'rxjs/operators';
+// OBSERVABLE HELPERS
+import { of } from 'rxjs/observable/of';
+// RXJS OBSERVABLE
+import { ofType } from 'redux-observable';
+// AJAX
+import { ajax } from 'rxjs/ajax';
+// REDUX ACTIONS 
+import {
+	GET_USER_REQUEST,
+	getUserSuccess,
+    getUserError,
+} from './userActions';
+
+// TODO: Hit actual endpoint for users
+
+const getUserEvent = (action$) => {
+	return action$.pipe(
+		ofType(GET_USER_REQUEST),
+		mergeMap(({ payload: { userId } }) =>
+			ajax.getJSON(`http://localhost:3100/users/${userId}/`).pipe(
```

Comment:
will fix this in next PR (add env variable for apiUrl)

Reply:
Sounds good!